programs/ssl/ssl_client2.c

```diff
@@ -3276,6 +3327,31 @@ int main( int argc, char *argv[] )
 #endif /* MBEDTLS_SSL_HANDSHAKE_WITH_PSK_ENABLED &&
           MBEDTLS_USE_PSA_CRYPTO */
 
+#if defined(MBEDTLS_KEY_EXCHANGE_ECJPAKE_ENABLED) && \
+    defined(MBEDTLS_USE_PSA_CRYPTO)
+    /*
+     * In case opaque keys it's the user responsibility to keep the key valid
```

Comment:
Typo I think: "In case opaque keys" (same in server).

Reply:
ops, that's true :/